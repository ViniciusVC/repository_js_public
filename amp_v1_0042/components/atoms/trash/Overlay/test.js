import React from "react";
import { shallow } from "enzyme";
import Component from "./index";

describe("Overlay >", () => {
  it("deve renderizar com o tema default", () => {
    const wrapper = shallow(<Component />);
    expect(wrapper.dive().getElement()).toMatchSnapshot();
  });
  it("deve renderizar com o rba do tema (1,1,1,1.2)", () => {
    const wrapper = shallow(<Component theme={{ rgba: "rgba(1,1,1,1.2)" }} />);
    expect(wrapper.dive().getElement()).toMatchSnapshot();
  });
});
