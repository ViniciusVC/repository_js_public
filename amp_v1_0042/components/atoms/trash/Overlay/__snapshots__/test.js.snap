// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overlay > deve renderizar com o rba do tema (1,1,1,1.2) 1`] = `
<aside
  className="overlay sc-bdVaJa QMdxG"
/>
`;

exports[`Overlay > deve renderizar com o tema default 1`] = `
<aside
  className="overlay sc-bdVaJa fOZlDJ"
/>
`;
